### ISSUE TITLE
Error: Could not create custom field...

### ISSUE THREAD

**Issue by bjendres** [CiviCooP/be.aivl.streetimport#12]
I'm currently getting the following exception:

```
Could not create custom field with name recruiting_organization_id in custom group 3, error from API CustomField Create: DB Error: unknown error
```

This happened while testing, but also when I tried to reinstall the extension.
@ErikHommel Could you fix this soon, or should I have a look?

**Comment by ErikHommel** [comment]
Will fix today

Verstuurd vanaf mijn iPad

Op 5 jun. 2015 om 10:01 heeft SYSTOPIA <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:

I'm currently getting the following exception:

Could not create custom field with name recruiting_organization_id in custom group 3, error from API CustomField Create: DB Error: unknown error

This happened while testing, but also when I tried to reinstall the extension.
@ErikHommelhttps://github.com/ErikHommel Could you fix this soon, or should I have a look?

**Comment by ErikHommel** [comment]
Custom Field requires specific group to be present, group now created in Config. Also, API Group Create add group_id to name of group, which is crap when creating a unique group in many installs. Added temp hack (update name in civicrm_group) and will raise core issue if there is not one yet.
This issue now fixed.

**Comment by ErikHommel** [comment]
Core bug https://issues.civicrm.org/jira/browse/CRM-14062, will be solved once AIVL have upgraded to 4.4.4. Test with 4.4.4> install and check for version in hack.

**Comment by ErikHommel** [comment]
Tested in 4.4.13, resolved. Added check on version < 4.5 (to be sure to be sure)
